src/test/java/org/beanpod/switchboard/controller/EncoderControllerTest.java

```diff
@@ -46,122 +40,109 @@
     private EncoderMapper encoderMapper;
 
     //stubbed Objects
-    static private DeviceEntity device1, device2;
-    static private EncoderEntity encoder1, encoder2;
+    static private DeviceEntity device;
+    static private DeviceDTO deviceDto;
+    static private EncoderEntity encoder;
+    static private EncoderDTO encoderDTO;
     static private List<EncoderEntity> listOfEncoders;
 
+    @BeforeAll
+    static void encoderFixture() throws ParseException {
```

Comment:
Could we remove this `throws ParseException` by using `SneakyThrows` in the fixtures?

Reply:
yes, fixed